lib/src/input_validation/pattern.dart

```diff
@@ -5,4 +5,5 @@ class Pattern {
   static final validNoPlusMinusOperator = RegExp(r'^[)^,*\/=]$');
   static final validBasicOperator = RegExp(r'^[+\-\/*^]$');
   static final validCommaBasicOperator = RegExp(r'^[,+\-\/*^]$');
+  static final validNumber = RegExp(r'[0-9]+');
```

Comment:
Don't think this is needed because it doesn't accept numbers with decimals, negatives, or pi or e. That's what `validOperand` is for

Reply:
removing